app/src/main/res/values/strings.xml

```diff
@@ -193,8 +193,10 @@
     <string name="enable_clipboard_history">Enable clipboard history</string>
     <!-- Description for enabling/disabling clipboard history mentioning that if disabled, clipboard content is pasted -->
     <string name="enable_clipboard_history_summary">If disabled, clipboard key will paste clipboard content if any</string>
-    <!-- Preferences item for enabling clipboard history -->
+    <!-- Preferences item for controlling clipboard history retention time -->
     <string name="clipboard_history_retention_time">History retention time</string>
+    <!-- Preferences item for showing unpinned clipboard items first -->
+    <string name="clipboard_history_unpinned_first">Show unpinned items on top</string>
```

Comment:
Maybe switch to the setting being about *pinned* items instead of ***un**pinned*? The _un_ feels unsuitable in the setting name.

Reply:
Yes, then default would be on